src/connector.rs

```diff
@@ -91,13 +93,15 @@ impl EthConnectorContract {
             contract: contract_data,
             ft,
         }
-        .save_contract();
+        .save_ft_contract();
     }
 
     /// Parse event message data for tokens
     fn parse_event_message(&self, message: &str) -> TokenMessageData {
+        // TODO: Why not use borsh here? Does message guarantee to not have `:` as part of inner fields
```

Comment:
Because that's from Ethereum. So it should be a simple string.
":" is the separator for two cases:
1. Near account transfer (NEAR accounts can't contain ":")
2. Eth account transfer

Reply:
We discussed that for the future we might extend the protocol for having several colons for the data separation. So as for me, as we parse&check these parts of the string, we should be fine. What issue do you see here?